src/test/java/seedu/address/model/tag/TagTest.java

```diff
@@ -26,10 +28,21 @@ public void isValidTagName() {
     @Test
     public void capitaliseTest() {
         Tag tag = new Tag("software engineer");
-        assert(tag.capitalise().equals("Software Engineer"));
+        assertTrue(tag.capitalise().equals("Software Engineer"));
 
         tag = new Tag("softWare EnginEEr");
-        assert(tag.capitalise().equals("Software Engineer"));
+        assertTrue(tag.capitalise().equals("Software Engineer"));
+
+    }
+
+    @Test
+    public void equals() {
```

Comment:
do you think it would be better to add more? something like the NameTest

         Tag tag = new Tag("Software Engineer");

        // same values -> returns true
        assertTrue(tag.equals(new Tag("Software Engineer")));

        // same object -> returns true
        assertTrue(tag.equals(tag));

        // null -> returns false
        assertFalse(tag.equals(null));

        // different types -> returns false
        assertFalse(tag.equals(5.0f));

        // different values -> returns false
        assertFalse(tag.equals(new Tag("Not Software Engineer")));

Reply:
Done! Thank you for the suggestion!